queries/ecma/locals.scm

```diff
@@ -26,6 +26,10 @@
   ((identifier) @definition.var)
    (#set! definition.var.scope parent))
 
+(method_definition
+  ((property_identifier) @definition.var)
+   (#set! definition.var.scope parent))
```

Comment:
isn't this a for method parameters`

Reply:
Which bit, sorry?

Reply:
`definition.var` and `definition.var.scope` ?

Reply:
What is property_identifier when used inside a method

Reply:
`property_identifier` seems to be the `identifier` equivalent for `method_definition`. I didn't find an associated `identifier` in the Tree-Sitter playground for methods, only functions. I wasn't clear on the difference/use of `definition.var.scope` from the documentation so added it to match the `function_declaration` query

Reply:
but what would `property_identifier` be in a code example?

Reply:
Yes, it would be the function name. I suspected that they should both be `@definition.function` as well, but wasn't sure.

```javascript
class Foo {
  bar() {
//^^^ property_identifier
  }
}
```

Reply:
can you change it also for function?